Consolidated checklist info (#222)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/install-aws.adoc

and networking requirements.

Your organization's operations, networking and security teams should review these requirements. See 
xref:install-prereqs.adoc[Anypoint Runtime Fabric Installation Prerequisites].
====

> == Before you Begin

Reviewer: ```suggestion
== Before You Begin
```

+
If you experience an error related to AWS authorization, ensure you're using the same terminal window for 
verifying the variables and running the Terraform command.
. Run the script.
+

> The Terraform script provisions the infrastructure and runs the installation script on each VM. When complete, the Runtime Fabric update to Active status in Runtime Manager.

Reviewer: ```suggestion
The Terraform script provisions the infrastructure and runs the installation script on each VM. When the installation is completed, Runtime Fabric updates to `Active` status in Runtime Manager.
```

+
[NOTE]

> This step installs Runtime Fabric across all servers to form a cluster. It may take 15-25 minutes or longer to complete.

Reviewer: ```suggestion
This step installs Runtime Fabric across all servers to form a cluster. It might take 15 to 25 minutes or longer to finish.
```

=== Run Terraform Within Docker

> . Navigate to the `../rtf-install-scripts/` directory. You must run Terraform from this directory. When running the `dir` or `ls` command, you should see the `aws` directory listed along with other directories (`azure`, `manual`, etc).

Reviewer: GLOBAL formatting for steps, per style ref:
```suggestion
. Navigate to the `../rtf-install-scripts/` directory. 
+
+ You must run Terraform from this directory. When running the `dir` or `ls` command, you should see the `aws` directory listed along with other directories (`azure`, `manual`, and so on).
```

> . Initialize Terraform (this only has to be performed once):

Reviewer: ```suggestion
. Initialize Terraform (this is performed only once):
```

  -state=tf-data/rtf.tfstate
----

> . Modify this using the data in the environment variables tables previously provided.

Reviewer: ```suggestion
. Modify this syntax by using the data in the environment variables tables previously provided.
```

  -state=tf-data/rtf.tfstate
----
+

> * The Terraform script provisions the infrastructure and runs the installation script on each VM. When complete, 

Reviewer: ```suggestion
* The Terraform script provisions the infrastructure and runs the installation script on each VM. When the installation is completed, 
```

> the Runtime Fabric is shown in *Active* status in Runtime Manager.

Reviewer: ```suggestion
Runtime Fabric is shown in *Active* status in Runtime Manager.
```
I earlier changed Active to `Active`, believing that this was a syntax display, but if it is UI, then please change that edit to *Active*.

+
[NOTE]

> This step installs Runtime Fabric across all servers to form a cluster. It may take 15-25 minutes or longer to complete.

Reviewer: ```suggestion
This step installs Runtime Fabric across all servers to form a cluster, which might take 15 to 25 minutes or longer to be completed.
```
